tests/TestCIT_KCI.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+
+import numpy as np
+
+import causallearn.utils.cit as cit
+
+class TestCIT_KCI(unittest.TestCase):
+    def test_Gaussian(self):
+        X = np.random.randn(300, 1)
+        X1 = np.random.randn(300, 1)
```

Comment:
`np.random.randn` samples from standard normal distribution. To verify KCI's performance, how about more non-Gaussian distributions?